src/ngraph/runtime/gpu/gpu_backend.cpp

```diff
@@ -67,20 +67,62 @@ bool runtime::gpu::GPU_Backend::call(shared_ptr<Function> func,
 
     validate_call(func, outputs, inputs);
 
-    auto it = m_function_map.find(func);
-    if (it == m_function_map.end())
+    FunctionInstance& instance = m_function_map[func];
```

Comment:
Pretty sure calling `m_function_map::operator[](func)` will default construct an entry in m_function_map if one does not exist. Why not use find?

Reply:
Copied from CPU code. Since you need to create one anyway if it's not exist. I think code is more clean in this way.

Reply:
Another reason is `enable_performance_data` can also create an key in `m_function_map`, so could not use find itself to determine if it's already compile.